modyn/protos/selector.proto

```diff
@@ -51,6 +51,7 @@ message GetSamplesRequest {
   int32 trigger_id = 2;
   int32 partition_id = 3;
   int32 worker_id = 4;
+  optional bool shuffle = 5;
```

Comment:
Makes the changes a bit less intrusive since we don't need to update the other codepaths using this function. If you want to, I can make it non-optional, but I thought it's okay to default to not shuffling for the gRPC path not related to training, while for training we want to make sure users have a decision